repo/packages/D/dcos-autoscaler/0/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "packagingVersion": "3.0",
+  "name": "dcos-autoscaler",
+  "version": "1.0.0",
+  "maintainer": "ahmadhassan612@gmail.com",
+  "description": "Adds autoscaling to your application based on cpu and memory metrics",
+  "scm": "https://github.com/Ahmadposten/DC-OS-Marathon-autoscaling",
+  "framework": true,
+  "tags": ["autoscaling", "mesosphere", "dcos"],
```

Comment:
`mesosphere` tag is reserved for packages developed by Mesosphere. Please remove it.